docs/04-operation-guides/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Did not receive an event - basic diagnostics
+---
+
+## Symptom
+
+You publish an event but the event is not received by the subscriber.
+
+## Remedy
+
+Follow these steps to detect the source of the problem:
+
+1. Check the eventing backend CRD, if eventingReady is true. Run the command:
+   
+   ```bash
+   kubectl -n kyma-system get eventingBackend
+   ```
+   If eventingReady is false, check the exact reason of the error in the status of the CRD by running the command:
```

Comment:
Should we use the all upper case version to relate it to the shell output ? 🤔 

```suggestion
   If EVENTINGREADY is false, check the exact reason of the error in the status of the CRD by running the command:
```
```
kubectl -n kyma-system get eventingBackend
NAME               BACKEND   EVENTINGREADY   SUBSCRIPTIONCONTROLLERREADY   PUBLISHERPROXYREADY
eventing-backend   BEB       true            true                          true
```

Reply:
You're right!